homeassistant/auth/__init__.py

```diff
@@ -158,6 +160,11 @@ def __init__(
         self._mfa_modules = mfa_modules
         self.login_flow = AuthManagerFlowManager(hass, self)
         self._revoke_callbacks: dict[str, list[CALLBACK_TYPE]] = {}
+        self._expire_callbacks: dict[str, CALLBACK_TYPE] = {}
+        self._expire_callback: CALLBACK_TYPE | None = None
+
+        hass.async_add_job(self.async_track_next_refresh_token_expiration)
+        hass.async_add_shutdown_job(HassJob(self.async_cancel_expiration_schedule))
```

Comment:
I changed this to `async_add_job` but it really should happen in another function so to ensure auth store is loaded first before this is called

Reply:
done with [move scheduler init to async_setup 3350077](https://github.com/home-assistant/core/pull/108428/commits/335007754adaabfade1b64c89a0fdb72f7e7c76a) from above